applications/tari_console_wallet/src/ui/state/app_state.rs

```diff
@@ -405,6 +485,14 @@ impl AppState {
         &self.cached_data.contacts[start..end]
     }
 
+    pub fn get_burnt_proofs_slice(&self, start: usize, end: usize) -> &[UiBurntProof] {
+        if self.cached_data.burnt_proofs.is_empty() || start > end || end > self.cached_data.burnt_proofs.len() {
```

Comment:
```suggestion
        if self.cached_data.burnt_proofs.is_empty() || start => end || end > self.cached_data.burnt_proofs.len() {
```
if start == end and end == len, this will panic